documentation/suggested-project-structure.md

```diff
@@ -0,0 +1,97 @@
+### Suggested Project Structure
+
+ * `server.js` is the main entry point.
+ * Think of the `resources` folder as your `routes`.
```

Comment:
If these are synonymous then why aren't they named as such? I think these two lines might be superfluous. Could you instead describe what they do and allow the reader to make the connection?